### ISSUE TITLE
Pass Total Deductible Amount to payment processor on Checkout page via Credit Cared charge URL

### ISSUE THREAD

**Issue by johnfine** [davidhabib/Auctions-for-Salesforce#127]
For orgs that want to save time/cost by relying on their integrated payment processor's automatic email receipting of auction purchases, rather than sending the paper or pdf receipt to each guest....

If A4SF offered the total Taxable Value (OR total FMV) as a parameter to the Credit Card Charge URL, then the integrated payment processor could provide this info on their receipt.  C&P already has the proper parameter to do this.

Auction purchasers need a tax receipt that includes this info.  Currently, either the receipt pdf must be emailed or paper mailed to each purchaser manually, or this info must be provided manually to the payment processor's receipting system.

Another solution to this issue would be to provide emailing of auctions receipts on the Receipts tab.

**Comment by johnfine** [comment]
Here are the details for Click and Pledge:

C&P can accept the URL parameter:

d(1..n): Tax deductible portion (optional)

If A4SF provided the total Auction Deductible Amount as a parameter (say, {7} in the parameter list), then the URL string could look like:

......&up1={2}&d1={7}.....

This would allow the A4SF user to charge the donor's cc from the A4SF checkout tab, in the Payment Method section, and for C&P to send a receipt to the donor that lists both the total amount charged to the donor's cc *and also* the total deductible amount.

**Comment by davidhabib** [comment]
@johnfine I'm looking into adding this parameter.  If I make a package with this feature, are you willing to install it and test it out with C&P?  let me know.  thanks.

**Comment by johnfine** [comment]
Absolutely! Our big yearly fundraiser is in two weeks, and we'll be using A4SF and C&P, so if you can get to it in the next week or so, that will give me time to install the package, customize my custom setting CreditCardChargeURLFormula to pass the info to C&P, customize the C&P receipt to use it, and test in a real world scenario. Feel free to reach out direct to me if you have any details to discuss. Many thanks for considering.
